aws/amazon-chime-sdk-js
# Add meeting events ingestion

**Issue #:**
NA

**Description of changes:**
- Add event ingestion components, documentation, unit tests to ingest events to Amazon chime backend.
- Add integration test for now so that it can report events to ingestion URL and then later we will remove the check for this in demo so that all canaries can report it.

**Testing**

1. Have you successfully run `npm run build:release` locally? Yes.
2. How did you test these changes?
- Tested locally and by deploying the browser demo app.
- Run the browser demo app and join a meeting.
- Select various audio inputs, video inputs, trigger re-connection for 5s and connect back and then leave meeting.
- All meeting events will be sent to the ingestion backend and you can monitor the request in browser devtools.

3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it?
The browser demo app.
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
I have added public API which have been reviewed on the private repo.
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No.


By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
